plugins/search/src/components/SearchResult/SearchResult.tsx

```diff
@@ -65,9 +66,14 @@ type Filters = {
 // TODO: move out column to make the search result component more generic
 const columns: TableColumn[] = [
   {
-    title: 'Component Id',
+    title: 'Name',
     field: 'name',
     highlight: true,
+    render: (entity: Partial<Result>) => (
+      <Link to={`/catalog/${entity.namespace}/${entity.kind}/${entity.name}`}>
```

Comment:
What do you think of encapsulating the URL/path in the `apis.ts` file above, rather than here?  So instead of adding `namespace` and pulling it out here, we add `url` and set `<Link to={entity.url} ... />`

Reply:
like it!

Reply:
@iamEAP done